scraping/spiders/poland_rjps.py

```diff
@@ -77,6 +77,10 @@ def _get_name(self, response):
         raw = response.css('h3.nazwa a::text').get()
         return self._clean_spaces(raw)
 
+    def _get_organization(self, response):
+        raw = response.css('div[title="Typ jednostki"] > span::text').get()
```

Comment:
What does this mean? Is this really a proper name of an organization or just a type? If it's just a type it's not right to put into this field

Reply:
`Typ jednostki` means Unit type,
then I think we can ignore this PR.